echo/server/dembrane/database.py

```
@@ -1,6 +1,6 @@
 # this is not upto date. switched to directus for a better life
 from enum import Enum
-from typing import Any, List, Optional, Annotated, Generator
+from typing import Any, Dict, List, Optional, Annotated, Generator
```

Ruff (0.8.2)

`typing.Dict` imported but unused

Remove unused import: `typing.Dict`

(F401)